packaging/ubuntu-16.04/control

```diff
@@ -112,3 +112,12 @@ Section: oldlibs
 Pre-Depends: dpkg (>= 1.15.7.2)
 Description: Transitional package for snapd
  This is a transitional dummy package. It can safely be removed.
+
+Package: snapd-xdg-open
```

Comment:
CC @mvo5 - I think we just want to have this in the main snapd package and have snapd-xdg-open as an empty transitional package.